src/Provider/AbstractProvider.php

```diff
@@ -24,16 +24,6 @@
     const ACCESS_TOKEN_UID = null;
 
     /**
-     * @var string HTTP method used to fetch access tokens.
```

Comment:
Strings are not constants. Comparing strings to strings is messy, whereas using:

```php
if ($thing === Foo::CONST) { ... }
```

is much nicer and does not require PHP to create new temporary values in memory.